python/covid_web/covid_web/server.py

```diff
@@ -63,13 +62,22 @@ def _create_rest(server: Flask, data_dir: Path, prediction_dir: Path):
     def covid19_country_data(country):
         return rest.get_country_data(country)
 
-    @server.route("/covid19/rest/predictions/list/")
+    @server.route("/covid19/rest/predictions/list")
     def covid19_available_predictions():
         return rest.get_available_predictions()
 
-    @server.route("/covid19/rest/predictions/data/<date>/<country>")
-    def covid19_get_specific_prediction(date, country):
-        return rest.get_specific_prediction(date, country)
+    @server.route("/covid19/rest/predictions/by_prediction/<prediction>")
```

Comment:
Aren't we getting rid of the Flask endpoints?

Reply:
Same as previous.